Rom.py

```diff
@@ -669,6 +669,8 @@ def patch_rom(world, player, rom):
     
     if difficulty.progressive_bow_limit < 2 and world.swords == 'swordless':
         rom.write_bytes(0x180098, [2, overflow_replacement])
+
+    if world.difficulty_adjustments in ['hard', 'expert']:
```

Comment:
You may need to add this line:
`rom.write_byte(0x180182, 0x00)  # Don't auto equip silvers on pickup`